mediasms.go

```diff
@@ -0,0 +1,499 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// SendRequest struct
+type SendRequest struct {
+	Smsid         string `json:"smsid"`
+	Smstitle      string `json:"smstitle"`
+	Smstext       string `json:"smstext"`
+	Mobilenumber  string `json:"mobilenumber"`
+	Originalurl   string `json:"originalurl"`
+	Originalurl2  string `json:"originalurl2"`
+	Originalurl3  string `json:"originalurl3"`
+	Originalurl4  string `json:"originalurl4"`
+	Status        string `json:"status"`
+	Returnsms     string `json:"returnsms"`
+	Waitreturnsms string `json:"waitreturnsms"`
+	Type          string `json:"type"`
+}
+
+// APIResponse struct
+type APIResponse struct {
+	StatusCode  string
+	Name        string
+	Description string
+}
+
+// ResultCode from media4u
+type ResultCode struct {
+	Result string `json:"result"`
+}
+
+// WebHook from media4u
+type WebHook struct {
+	Mobilenumber      string
+	Status            string
+	Smsid             string
+	Returnsms         string
+	Raitreturnsms     string
+	Returnsmsdatetime string
+	Replyid           string
+	Senderid          string
+}
```

Comment:
Do these structs need to be exposed?

Reply:
`SendRequest` and `WebHook` should definitely be usable by library consumers I think. However, I think `ResultCode` and `APIResponse` can likely be made private.

Reply:
Actually, with the way it is structured now, it does have to be exported from its file, so it can be used elsewhere.